src/seedu/addressbook/parser/Parser.java

```diff
@@ -75,33 +62,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
+                return prepareAdd(arguments);
+
+            case DeleteCommand.COMMAND_WORD:
+                return prepareDelete(arguments);
 
-        case DeleteCommand.COMMAND_WORD:
-            return prepareDelete(arguments);
+            case ClearCommand.COMMAND_WORD:
+                return new ClearCommand();
 
-        case ClearCommand.COMMAND_WORD:
-            return new ClearCommand();
+            case FindCommand.COMMAND_WORD:
+                return prepareFind(arguments);
 
-        case FindCommand.COMMAND_WORD:
-            return prepareFind(arguments);
+            case ListCommand.COMMAND_WORD:
+                return new ListCommand();
 
-        case ListCommand.COMMAND_WORD:
-            return new ListCommand();
+            case ViewCommand.COMMAND_WORD:
+                return prepareView(arguments);
 
-        case ViewCommand.COMMAND_WORD:
-            return prepareView(arguments);
+            case ViewAllCommand.COMMAND_WORD:
+                return prepareViewAll(arguments);
 
-        case ViewAllCommand.COMMAND_WORD:
-            return prepareViewAll(arguments);
+            case ExitCommand.COMMAND_WORD:
+                return new ExitCommand();
 
-        case ExitCommand.COMMAND_WORD:
-            return new ExitCommand();
+            case SortCommand.COMMAND_WORD:
```

Comment:
When you add a new command, you should edit the user guide to reflect these changes.